src/layers/HandicapLayer/HandicapLayer.js

```diff
@@ -0,0 +1,107 @@
+import VectorLayer from 'react-spatial/layers/VectorLayer';
+import OLVectorLayer from 'ol/layer/Vector';
+import OLVectorSource from 'ol/source/Vector';
+import GeoJSON from 'ol/format/GeoJSON';
+import { Style, Circle, Fill } from 'ol/style';
+import CONF from '../../config/appConfig';
+
+/**
+ * Layer for visualizing Handicap Topic.
+ * @class
+ * @params {Object} options
+ * @inheritdoc
+ */
+class HandicapLayer extends VectorLayer {
+  constructor(options = {}) {
+    const olLayer = new OLVectorLayer({
+      style: (f, r) => this.style(f, r),
+      source: new OLVectorSource(),
+    });
+
+    super({
+      ...options,
+      olLayer,
+    });
+
+    this.url = `${CONF.cartaroUrl}handicap/items/`;
+
+    this.setVisible(this.visible);
+
+    this.showStuetzpunktbahnhof = options.showStuetzpunktbahnhof;
+
+    this.onClick(f => {
+      [this.clickedFeature] = f;
+      this.olLayer.changed();
+    });
+  }
+
+  /**
+   * Create Style from feature
+   * @param {ol.feature} feature
+   * @returns {Object|null}
+   */
+  style(feature) {
+    const style = [
+      new Style({
+        image: new Circle({
+          radius: 7,
+          fill: new Fill({
+            color: feature.get('stuetzpunktbahnhof') ? 'blue' : 'red',
+          }),
+        }),
+      }),
+    ];
+
+    if (feature === this.clickedFeature) {
+      style.unshift(
+        new Style({
+          image: new Circle({
+            radius: 10,
+            fill: new Fill({
+              color: 'rgba(0, 61, 133, 0.5)',
+            }),
+          }),
+        }),
+      );
+    }
+
+    return style;
+  }
+
+  /**
+   * Set visible
+   * @param {boolean} visible
+   * @param {boolean} [stopPropagationDown] Stops propagation down.
+   * @param {boolean} [stopPropagationUp] Stops propagation up.
+   * @param {boolean} [stopPropagationSiblings] Stops propagation toward siblings.
+   */
+  setVisible(
```

Comment:
We don't need to overwrite the `setVisible()` method if we use the VectorLayer like here: https://openlayers.org/en/latest/examples/vector-layer.html